source/plugin.mjs

```diff
@@ -0,0 +1,145 @@
+import { fileError, liftChildren, NotebookCell } from 'myst-common';
+import { remove } from 'unist-util-remove';
+
+const MODE = 'python';
```

Comment:
This likely will be an env variable so that you can `MODE = "python" && myst start` or similar.